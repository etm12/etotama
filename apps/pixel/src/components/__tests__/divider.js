import * as React from 'karet';
import { mount } from 'enzyme';
import Divider from '../divider';

describe('Divider', () => {
  test('mounts without crashing', () => {
    mount(<Divider />);
  });

  test('has not inadvertently changed', () => {
    expect(mount(<Divider />)).toMatchSnapshot();
  });
});
