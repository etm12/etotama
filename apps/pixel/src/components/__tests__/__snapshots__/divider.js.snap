// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider has not inadvertently changed 1`] = `
<Divider>
  <div
    className="c-divider c-divider--thin"
  />
</Divider>
`;
